src/example/allocate_module.c

```diff
@@ -21,7 +21,7 @@
 #endif
 #include <sys/types.h>
 
-#if UNIT_TESTING
+//#if UNIT_TESTING
```

Comment:
We need to keep this in place so that the wrapper for the allocator is only in place when testing.